src/main/java/edu/hm/hafner/analysis/parser/MentorParser.java

```diff
@@ -0,0 +1,143 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.LookaheadParser;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.util.LookaheadStream;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/**
+ * Parser for Mentor Graphics Modelsim/Questa Simulator.
+ *
+ * @author Derrick Gibelyou
+ */
+public class MentorParser extends LookaheadParser {
+
+    /**
+     * Matches the beginning of a Modelsim questa message "** [priority] : [The remainder of the message]".
+     * The first capture group is the priority, the second capture group is the rest of the message to be parsed later.
```

Comment:
Maybe it would make more sense to use named capturing groups? See https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/GradleErrorProneParser.java#L22 for an example

Reply:
I added the javadoc because one of the code checkers was complaining, but named groups are a good idea.